### PR TITLE
Add addon hodgepodge writeup

### PR DESCRIPTION
**Pull request by amyrlam**
## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [x] Check that all links work

### CODE REVIEW

File: content/the-ember-times-issue-178.md [hunk 1]
```diff
@@ -26,15 +28,14 @@ tags:
 
 ---
 
-## [2. Section title in sentence case 🐹](section-url)
+## Addon roundup 🙌
```
Comment (amyrlam) [inline content/the-ember-times-issue-178.md:31]: could also change to `Addon hodgepodge roundup`, if not too weird 😅

Comment (ijlee2) [reply content/the-ember-times-issue-178.md:31]: I think Addon roundup sounds fine. 🙂

File: content/the-ember-times-issue-178.md [hunk 2]
```diff
@@ -26,15 +28,15 @@ tags:
...
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+We wanted to highlight some recent happenings around the web! Check them out and give them a try! 😎
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+<!-- alex ignore easy -->
+* Do you want to use OAuth in your Ember app? [Sinan Keskin (@sinankeskin)](https://github.com/sinankeskin) wrote a new addon, [ember-oauther](https://github.com/sinankeskin/ember-oauther), which allows you to sign in and register a user with OAuth easily! OAuth is an open standard for **access delegation**, commonly used as a way to register or log into a web app via their Facebook, Google, Microsoft, etc. login.
```
Comment (ijlee2) [inline content/the-ember-times-issue-178.md:36]: optional:

```suggestion
* Do you want to use OAuth in your Ember app? [Sinan Keskin (@sinankeskin)](https://github.com/sinankeskin) wrote a new addon, [ember-oauther](https://github.com/sinankeskin/ember-oauther), which allows you to sign in and register a user with OAuth easily! OAuth is an open standard for **access delegation**, commonly used to help a user register or log in to a web app via their account for Facebook, Google, Microsoft, etc.
```

**Review by ijlee2 — APPROVED** [review]
Thanks for covering new releases of addons!

As an option, I think we can have multiple paragraphs in a bullet point if the subsequent paragraphs are indented with 4 additional spaces. This may be useful if you want to break down a large paragraph in a bullet point.

**Comment by ijlee2** [comment]
@amyrlam I'll go ahead and merge this PR so that I can work with the latest Markdown file. If you want to make further changes, can you edit the file in the destination branch?

File: content/the-ember-times-issue-178.md (hunk 2 above)
Comment (amyrlam) [reply content/the-ember-times-issue-178.md:36]: brought this over to the main branch!, feel free to commit edits anytime!

**Comment by amyrlam** [comment]
@ijlee2 many thanks for the review, i brought your oauth edits in to the main branch!

**Pull request merged**